File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L6/06/inout.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function hitungVol(ar: number, l: number): void {
  const vol = ar * l;
  console.log(`The volume of the cylinder is ${vol}`);
}

function hitungAr(rad: number): number {
  const ar = rad * rad * 3.14159;
  console.log(`The area is ${ar}`);
  return ar;
}

rl.question('Enter the radius and length of a cylinder: ', (input) => {
  const [rad, l] = input.split(' ').map(Number);
  hitungVol(hitungAr(rad), l);
  rl.close();
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L2/06/inout.py
import math

# Enter radius of the cylinder
radius = float(input("Enter the radius and length of a cylinder: "))
length = float(input())

area = radius * radius * math.pi
volume = area * length

print(f"The area is {area:.5f}")
print(f"The volume of the cylinder is {volume:.5f}")
